### DISCUSSION TITLE
"ALTER SYSTEM" command permission denied error

### DISCUSSION THREAD

**Discussion by ShanArosh** [supabase/supabase#18581] [Feature Requests]
I'm in a problem where I need to change timescale-db extension license to community license from apache license.
otherwise their core functionalities wouldn't work like ,
 - create materialized view

I tried running
 ```
ALTER SYSTEM SET timescaledb.license_key = 'timescale'; 
```
and got permission denied error 

```
ERROR:  42501: permission denied to set parameter "timescaledb.license_key"
```

**Comment by saltcod** [comment]
closing in favour of https://github.com/supabase/supabase/issues/18585
